website/content/docs/operators/heron-tracker.md

```diff
@@ -51,15 +51,25 @@ statemgrs:
     name: "local"
     rootpath: "~/.herondata/repository/state/local"
     tunnelhost: "localhost"
-  -
-    type: "zookeeper"
-    name: "localzk"
-    hostport: "localhost:2181"
-    rootpath: "/heron/cluster"
-    tunnelhost: "localhost"
+#
+# To use 'localzk', launch a zookeeper server locally
+# and create the following path:
+#  *. /heron/topologies
+#
+#  -
+#    type: "zookeeper"
+#    name: "localzk"
+#    hostport: "localhost:2181"
+#    rootpath: "/heron"
+#    tunnelhost: "localhost" -
 ```
 
-Note that topologies from all the state managers would be read.
+Topologies from all the state managers would be read and can be queried from
+Tracker.
+
+Note that Tracker does not create any zookeeper nodes itself. It is a readonly
```

Comment:
A few things:

* s/zookeeper/ZooKeeper
* s/readonly/read-only
* without creating a `/topologies` node (instead of just "without creating `/topologies` node)
* s/rootpath/root path